benefit/data/sheets/FixedLists/FixedLists.json

```diff
@@ -196,6 +196,7 @@
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "adjournmentNoticeIssued", "ListElement": "Adjournment notice issued"},
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "hearingRecordingProcessed", "ListElement": "Hearing recording processed"},
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "hearingRecordingActioned", "ListElement": "Hearing recording actioned"},
+  {"LiveFrom": "13/05/2022", "ID": "FL_dwpStates", "ListElementCode": "hearingDateIssued", "ListElement": "Hearing Date Issued"},
```

Comment:
Does this work in and itself? There's an enum called DwpState which I believe lines up with this. We should add the new entry in to keep things consistent.

Reply:
@dattipoeHMCTS I have updated sscsCommon and related service's sscsCommon version. Thank you for reminding.